### PR TITLE
fix link to old website repo

### PR DESCRIPTION
**Pull request by brunob**

### CODE REVIEW

File: content/new-website.rst [hunk 1]
```diff
@@ -11,4 +11,4 @@ The libgd.org has a new website.  We have built the new site in
 view the sources used to build this website in its own `git repository`_.
 
 .. _Pelican: http://getpelican.com
-.. _git repository: https://github.org/libgd/website
+.. _git repository: https://github.com/libgd/oldwebsite
```
Comment (vapier) [inline content/new-website.rst]: it should be pointing to this repo -- `https://github.com/libgd/libgd.github.io`

**Review by vapier — CHANGES_REQUESTED** [review]

File: content/new-website.rst (hunk 1 above)
Comment (brunob) [reply content/new-website.rst]: Ha, my bad, i've put this cause actual link redirect to https://github.com/libgd/oldwebsite

Comment (vapier) [reply content/new-website.rst]: that was the correct site previously (and when this news entry was written), but we've consolidated the logic

**Comment by vapier** [comment]
thx bud

**Comment by brunob** [comment]
u're welcome :)

**Pull request merged**